### ISSUE TITLE
Default elements are not accessible to VoiceOver

### ISSUE THREAD

**Issue by hapasa** [facebook/react-native#336]
For example, VoiceOver does not read the \<text\> element contents.
So applications done with react-native are not accessible to blind or other VoiceOver users.

**Comment by gabro** [comment]
Some discussion going on in #342

**Comment by vjeux** [comment]
Closing in favor of #342
